src/evergreen.ts

```diff
@@ -125,6 +125,18 @@ export class client {
         this.getUIResource(callback, resource, params);
     }
 
+    /**
+     * Gets logs of a particular type for a particular task/
+     * @param callback - function to process the response
+     * @param id - identifier for the task whose logs we want
+     * @param type - type of log to return
+     * @returns nothing
+     */
+    public getLogs(callback: request.RequestCallback, id?: string, type?: string) {
```

Comment:
I think this also needs to take the execution number as a parameter, which will replace what you have as 0 below

Reply:
Good to know, I hardcoded a zero originally because I had no clue what that number was supposed to represent. It's now included as a parameter.